addons/website_forum/models/forum.py

```diff
@@ -126,6 +126,8 @@ def _get_default_faq(self):
     karma_editor = fields.Integer(string='Editor Features: image and links',
                                   default=30, oldname='karma_editor_link_files')
     karma_user_bio = fields.Integer(string='Display detailed user biography', default=750)
+    karma_post = fields.Integer(string='Ask questions without validation', default=100)
```

Comment:
What is the different with the karma_ask and karma_answer fields? Is it anything preventing from using those fields ?